### ISSUE TITLE
README.md and extra files

### ISSUE THREAD

**Issue by Jmainguy** [Tgar13/Pizza-Ordering-Chatbox#1]
this repo and https://github.com/Tgar13/CIS110-Course-Project look really cool. I enjoyed playing with the dog story in course-project.

if you could include a README.md with instructions on how to interact with the code in each repo, it would be really helpful to people coming across these.

Also, appears in both repos to be multiple versions of the file. You can remove all but the latest file, as they will be in git history, there is no need to keep them in the repo, where they can be confusing.

I also recommend having the python files end in .py to make it clear they are python files.

Really fantastic work, enjoyed both repos. Thank you for creating these

**Comment by Tgar13** [comment]
Hi! Thank you so much for your input. The dog story is definitely my favorite as I based it off my own German Shepherd. Each repository now has a README.md file. I also deleted the duplicate files and left the final output alone. Although, I will have numerous revisions in my latest repository, CIS 261, as it is ongoing and necessary per professor directions. In CIS 110, I added the final.py code. I've also included a write-up that describes the difficulties I encountered while creating this narrative. Have a wonderful day!
